.github/workflows/_build_base.yaml

```
@@ -57,8 +62,41 @@ permissions:
 
 jobs:
 
+  nccl-version:
+    runs-on: ubuntu-22.04
+    outputs:
+      JAX_NCCL_VERSION: ${{ steps.get-nccl-version.outputs.JAX_NCCL_VERSION }}
+      JAX_LIBNCCL_PACKAGE: ${{ steps.get-nccl-version.outputs.JAX_LIBNCCL_PACKAGE }}
+    steps:
+      - name: Print environment variables
+        run: env
+      - name: Check out the repository under ${GITHUB_WORKSPACE}
+        uses: actions/checkout@v4
+
+      - name: Get NCCL version
+        id: get-nccl-version
+        shell: bash -x -e {0}
+        run: |
+          JAX_LIBNCCL_PACKAGE=${{ inputs.JAX_LIBNCCL_PACKAGE }}
+          if [[ -z $JAX_LIBNCCL_PACKAGE ]]; then
+            BASE_IMAGE=${{ inputs.BASE_IMAGE }}
+            if [[ $BASE_IMAGE == latest ]]; then
+              BASE_IMAGE=$(cat .github/container/Dockerfile.base | sed -n "s/^ARG BASE_IMAGE=\(.*\)$/\1/p")
+            fi
+
+            # try to get NCCL version from provided BASE_IMAGE of ARM64-arch
+            source .github/workflows/scripts/get_remote_env.sh
+            JAX_LIBNCCL_PACKAGE=$(get_remote_env ${BASE_IMAGE:-ubuntu} linux amd64 | jq -r '.[]' | egrep '^NV_LIBNCCL_PACKAGE')
+            JAX_NCCL_VERSION=$(get_remote_env ${BASE_IMAGE:-ubuntu} linux amd64 | jq -r '.[]' | egrep '^NCCL_VERSION=' | cut -d= -f2-)
```

Is `ubuntu` the right default? Does the ubuntu base even have NCCL installed? I feel like there should be a check like
```
if [[ -z "$BASE_IMAGE" ]]; then
  echo "Need to pass non-empty BASE_IMAGE variable"
  exit 1
fi
```
earlier (maybe above line 83)

---

```suggestion
            JAX_LIBNCCL_PACKAGE=$(get_remote_env $BASE_IMAGE linux amd64 | jq -r '.[]' | egrep '^NV_LIBNCCL_PACKAGE')
            JAX_NCCL_VERSION=$(get_remote_env $BASE_IMAGE linux amd64 | jq -r '.[]' | egrep '^NCCL_VERSION=' | cut -d= -f2-)
```

How about removing this too since `ubuntu:latest` doesn't seem to have what we need in the image?

---

I removed it, but forgot to push. Done.